modules/viz/include/opencv2/viz/types.hpp

```diff
@@ -115,27 +114,87 @@ namespace cv
             static Color not_set();
         };
 
+        struct CV_EXPORTS_W_SIMPLE PyColor
```

Comment:
We should not have Python-specific classes in public C++ header.

Please move this to `viz/misc/python/python_bridge.hpp` (see gapi module for example)

Reply:
I need PyColor viz3d.hpp

 If I move PyColor defintion in python_bridge.hpp then it will generate error